azkaban-db/src/main/java/azkaban/db/DatabaseTransOperator.java

```diff
@@ -30,14 +30,14 @@
  */
 public interface DatabaseTransOperator {
```

Comment:
Oh. I see. So `DatabaseOperator` is a single query runner. `DatabaseTransOperator` is a multi query runner? 

nit: The name confused me a bit. Technically both are doing transactions. I'll let you decide on this but I think `SingleQueryRunner` and `MultiQueryRunner` would be more self explanatory - since we are already using the `QueryRunner` library. 

And I also like the comments that you added. It helps!